### ISSUE TITLE
iterm not working

### ISSUE THREAD

**Issue by reagle** [wbond/sublime_terminal#128]
Using `iterm 3.0.4` and `subl 3114`. When I add the iTerm.sh JSON as indicated on project page, nothing happens on invocation. If I remove `iTerm.sh` and continue with the empty string, Terminal will be used and works.

**Comment by twolfson** [comment]
This is a duplicate of #89. Please see #89 for the current workarounds

**Comment by reagle** [comment]
FYI: I did see that, but I'm not using a nightly and #89 is a pretty long and complicated discussion. It's not clear to me what the latest, recommended workaround is? Maybe the documentation for this plugin should be updated, at least to point to #89?

**Comment by twolfson** [comment]
I believe the workaround is to use the script from #98 and add it manually as a new script a la:

https://github.com/wbond/sublime_terminal/tree/1.12.0#cmder-on-windows

I concur that this is an issue that's been open for too long. I think we are still waiting on a resolution from @wbond though =/

https://github.com/wbond/sublime_terminal/pull/98#issuecomment-149056701
